### ISSUE TITLE
Jetpacks dont come stocked with materials after crafting 

### ISSUE THREAD

**Issue by GTink911** [zorkind/Hellion-Rescue-Project#53]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Jetpacks do not come stocked with materials immediately after crafting.

**Describe alternatives you've considered**
None.

**Additional context**
None.

**Comment by CheeseJedi** [comment]
Nor do welders, resource canisters etc. which is probably 'correct', IMO.

Filling a jetpack is currently trivial assuming you have the required resources since ZG added a suggestion I put in for a 'fill' button in the cargo panel some time back - until then there was only an 'empty' button.

This suggestion _could_ be implemented by altering the jetpacks crafting recipes to include the required oxygen and nitro and setting the fill levels of the jetpack tanks to 100% when spawned, although I believe this **reduces player choice** which makes it a bad thing in my book.

**Comment by GTink911** [comment]
I agree with the reduces player choice part, but the refill button has been implemented.
